### PR TITLE
Allow root control of breadcrumb

### PR DESCRIPTION
**Pull request by wraeth-eth**

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #212** <a href="https://app.graphite.dev/github/pr/peeldao/juicecrowd/212?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #213** <a href="https://app.graphite.dev/github/pr/peeldao/juicecrowd/213?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #214** <a href="https://app.graphite.dev/github/pr/peeldao/juicecrowd/214?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #215** <a href="https://app.graphite.dev/github/pr/peeldao/juicecrowd/215?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/peeldao/juicecrowd/215?utm_source=stack-comment).

File: src/components/Breadcrumbs.tsx [hunk 1]
```diff
@@ -1,35 +1,69 @@
...
 
   return (
     <div className={twMerge('flex text-sm text-gray-700', className)}>
-      <Link href="/">Home</Link>
+      <Link href={rootPath}>{root}</Link>
+
       {crumbs.slice(0, -1).map((crumb, i) => (
         <React.Fragment key={crumb}>
-          <span className="mx-2">/</span>
+          <span className="mx-2">{delimiter}</span>
```
Comment (johnnyd-eth) [inline src/components/Breadcrumbs.tsx:58]: Noiicee

File: src/components/Breadcrumbs.tsx [hunk 2]
```diff
@@ -1,35 +1,69 @@
...
+      .map(decodeURIComponent)
+      .map(crumb => {
+        return crumb[0].toUpperCase() + crumb.slice(1)
+      })
+  }, [path, rootKey])
+
+  const root = useMemo(() => {
+    if (!rootKey) return 'Home'
+
+    return rootKey[0].toUpperCase() + rootKey.slice(1)
```
Comment (johnnyd-eth) [inline src/components/Breadcrumbs.tsx:43]: **Question:** worth a `rootKey[0]?.toUpperCase()`?

**Review by johnnyd-eth — APPROVED** [review]
One comment but LGTM

**Comment by wraeth-eth** [comment]
### Merge activity

* **Nov 17, 6:31 PM**: @wraeth-eth started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/peeldao/juicecrowd/215).

**Pull request merged**